atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -208,6 +209,36 @@ def build_list(self, manifests):
                 ],
         }, indent=4)
 
+    def check_existing_vr_tag(self):
+        self.primary_images = get_primary_images(self.workflow)
+
+        if not self.primary_images:
+            return
+
+        vr_image = None
+        for image in self.primary_images:
+            if '-' in image.tag:
+                vr_image = image
+                break
+        if not vr_image:
+            return
+
+        should_fail = False
+        for registry_name, registry in self.registries.items():
+            pullspec = vr_image.copy()
+            pullspec.registry = registry_name
+            insecure = registry.get('insecure', False)
+            secret = registry.get('secret', None)
+
+            manifest_list = get_manifest_list(pullspec, registry_name, insecure, secret)
+
+            if manifest_list:
+                self.log.error("Primary tag already exists in registry: %s", pullspec)
+                should_fail = True
+
+        if should_fail:
+            raise RuntimeError("Primary tag already exists in registry")
```

Comment:
Do we want to print a list with the registries containing the tag here?

Reply:
I print it in the log.error above, for each occurence